app/views/admin/link_check_reports/_link_check_report.html.erb

```diff
@@ -17,9 +17,17 @@
     <section class="alert alert-warning">
       <h3 class="remove-top-margin">Broken links</h3>
       <p>We’ve found links in this document that may be broken:</p>
-      <ul>
+      <ul class="issue-list">
         <% report.broken_links.each do |link| %>
-          <li><%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %></li>
+          <li>
+            <%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %>
+            <% if link.suggested_fix %>
```

Comment:
I think we always want to see the problem summary, but only include the suggested fix part if there is a suggested fix.